cpp/src/gandiva/function_registry.h

```diff
@@ -17,7 +17,12 @@
 
 #pragma once
 
+#include <memory>
+#include <string>
 #include <vector>
+
+#include <arrow/buffer.h>
+#include <arrow/status.h>
```

Comment:
Should use regular quotes for Arrow includes:
```suggestion
#include "arrow/buffer.h"
#include "arrow/status.h"
```

Reply:
Fixed.